sdk/documenttranslation/Azure.AI.DocumentTranslation/tests/Infrastructure/DocumentTranslationLiveTestBase.cs

```diff
@@ -23,6 +23,20 @@ public DocumentTranslationLiveTestBase(bool isAsync, RecordedTestMode? mode = nu
             Sanitizer = new DocumentTranslationRecordedTestSanitizer();
         }
 
+        private static readonly List<TestDocument> oneTestDocuments = new()
+        {
+            new TestDocument("Document1.txt", "First english test document"),
+        };
+
+        private static readonly List<TestDocument> twoTestDocuments = new()
+        {
+            new TestDocument("Document1.txt", "First english test document"),
+            new TestDocument("File2.txt", "Second english test file"),
+        };
+
+        protected List<TestDocument> OneDocumentList = oneTestDocuments;
```

Comment:
These don't appear to be set below, so why not just make the `static readonly` fields `protected` and use those?